<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Materials</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../civiltools.qrc">
    <normaloff>:/creates/images/materials.svg</normaloff>:/creates/images/materials.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <widget class="QWidget" name="concrete">
      <attribute name="icon">
       <iconset resource="../../civiltools.qrc">
        <normaloff>:/creates/images/concrete-truck.svg</normaloff>:/creates/images/concrete-truck.svg</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>f'c</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="fc_spinbox">
           <property name="suffix">
            <string> MPa</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>20</number>
           </property>
           <property name="maximum">
            <number>70</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="fc_name">
           <property name="text">
            <string>C30</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Ec</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="ec1">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../civiltools.qrc">
              <normaloff>:/creates/images/ec1.svg</normaloff>:/creates/images/ec1.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="wc">
            <property name="suffix">
             <string> Kg/m3</string>
            </property>
            <property name="prefix">
             <string>Wc= </string>
            </property>
            <property name="minimum">
             <number>1440</number>
            </property>
            <property name="maximum">
             <number>2560</number>
            </property>
            <property name="singleStep">
             <number>50</number>
            </property>
            <property name="value">
             <number>2300</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="ec2">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../civiltools.qrc">
              <normaloff>:/creates/images/ec2.svg</normaloff>:/creates/images/ec2.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rebar">
      <attribute name="icon">
       <iconset resource="../../civiltools.qrc">
        <normaloff>:/creates/images/rebars.svg</normaloff>:/creates/images/rebars.svg</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="standard_rebars_groupbox">
         <property name="title">
          <string>Standard Rebars</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="2">
           <widget class="QSpinBox" name="s340fu_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="s400fu_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="s400_checkbox">
            <property name="text">
             <string>S400</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3" colspan="2">
           <widget class="QLineEdit" name="s340_name">
            <property name="text">
             <string>AII</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="s400fy_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>400</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="s340_checkbox">
            <property name="text">
             <string>S340</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QLineEdit" name="s400_name">
            <property name="text">
             <string>AIII</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="s420fy_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>420</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>fy (MPa)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="s420_checkbox">
            <property name="text">
             <string>S420</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="s340fy_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>340</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>fu (MPa)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="s420fu_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
          <item row="3" column="3" colspan="2">
           <widget class="QLineEdit" name="s420_name">
            <property name="text">
             <string>AIII-2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="s500_checkbox">
            <property name="text">
             <string>S500</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="s500fy_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="s500fu_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>650</number>
            </property>
           </widget>
          </item>
          <item row="4" column="3" colspan="2">
           <widget class="QLineEdit" name="s500_name">
            <property name="text">
             <string>AIV</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="other_rebars_groupbox">
         <property name="title">
          <string>Other Rebars</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>fy (MPa)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="other_fy_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>340</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="other_fu_spinbox">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>700</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3" colspan="2">
           <widget class="QLineEdit" name="other_name">
            <property name="text">
             <string>Rebar</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>fu (MPa)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" colspan="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="create_pushbutton">
     <property name="text">
      <string>Create</string>
     </property>
     <property name="icon">
      <iconset resource="../../civiltools.qrc">
       <normaloff>:/civiltools/images/etabs.png</normaloff>:/civiltools/images/etabs.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fc_spinbox</tabstop>
  <tabstop>fc_name</tabstop>
  <tabstop>ec1</tabstop>
  <tabstop>wc</tabstop>
  <tabstop>ec2</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
